### PR TITLE
Create handler for zoom urls, create_meeting and reschedule_meeting

### PR DESCRIPTION
**Pull request by fatcat2**
Methods and API calls were tested using Postman. DM me for the Zoom API key.

### CODE REVIEW

File: biit_server/meeting_handler.py [hunk 1]
```diff
@@ -749,6 +754,9 @@ def matchup(request, auth):
         match = [min_potential_id, min_user_id]
 
         random_id = str(uuid.uuid4())
+
+        zoom_info = create_meeting(in_a_week.timestamp(), 30, match)
```
Comment (lzhahn) [inline biit_server/meeting_handler.py:758]: How is zoom handled when a person wants to meet up in person? Is it still generated, but not sent?

File: biit_server/meeting_handler.py [hunk 2]
```diff
@@ -60,6 +61,8 @@ def meeting_post(request, auth):
         random.choice(string.ascii_letters + string.digits) for i in range(64)
     )
 
+    zoom_info = create_meeting(body["timestamp"], body["duration"], body["user_list"])
```
Comment (lzhahn) [inline biit_server/meeting_handler.py:64]: In-person may not need a zoom call

**Review by lzhahn — APPROVED** [review]
Looks good, take a look at the comments and it should be set to go!

**Review by stephend017 — APPROVED** [review]
im gonna trust you

**Pull request merged**